packages/clerk-js/src/ui/elements/Menu.tsx

```diff
@@ -72,10 +78,12 @@ const findMenuItem = (el: Element, siblingType: 'prev' | 'next', options?: { cou
   return sibling;
 };
 
-type MenuListProps = PropsOfComponent<typeof Col>;
+type MenuListProps = PropsOfComponent<typeof Col> & {
+  isPortal?: boolean;
```

Comment:
Why is this needed?

Reply:
Before, we always had the `<Menu/>` popover rendering in a Portal. However, for the `<ProfileSectionActionMenu/>`, we wanted to limit the width of the popover to match the size of the menu trigger. I added that flag to provide as an option to opt-out of Portal in cases where it's not desired, like the one I mentioned.

Reply:
Can we name it `portal` or `asPortal` ?